start_bundle_coupled.R

```diff
@@ -166,6 +156,7 @@ if (!identical(missing, character(0))) {
 }
 
 common <- intersect(rownames(settings_remind),rownames(scenarios_coupled))
+knownRefRuns <- apply(expand.grid(prefix_runname , common, "-rem-", seq(max_iterations)), 1, paste, collapse="")
```

Comment:
basically generated `c("Base-rem-1", "Base-rem-2", "Base-rem-3", "NDC-rem-1", "NDC-rem-2", "NDC-rem-3")` from `common = c("Base", "NDC")` and `max_iterations = 3`.